testing.rst

```diff
@@ -630,6 +630,10 @@ You can pass any
 :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\TestBrowserToken` object and
 stores in the session of the test client.
 
+.. note::
+
+    ``loginUser()`` method is not expected to work in all of the cases. As it relies on session-based authentication, it won't work if the routes you are trying to test are protected by a stateless firewall or if you are using a custom authenticator.
```

Comment:
This note is mentioning a problem, not a solution.

I'd rather have it tells something like:
```rest
Stateless firewalls cannot use ``loginUser()`` prior to requests by design.
Instead you should add the correct token (i.e. header) in each ``request()`` call.
```

I guess it would be even better if the phpdocs of the method could provide more context, or throw a logic exception when a firewall is stateless with a friendly message. ping @wouterj WDYT?

Reply:
Is there any action left for me to do? Or are we just waiting for comments/approbation from the reviewers?